draft-ietf-rats-corim.md

```diff
@@ -1217,14 +1217,18 @@ matched.
 ~~~
 
 The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
-Each `conditional-series-record` record is evaluated in the order it appears in the series array.
+
+Each `conditional-series-record` has measurements belonging to one or more measured elements, pertaining to the environment that appears in the stateful-environment record.
+
 The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the ACS.
-The first `conditional-series-record` that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted.
+The first `conditional-series-record` that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted. In order for a `conditional-series-record` to match, measurements pertaining to every measured element in the record MUST match.
+
 If none of the series conditions match an ACS Entry, the triple is not matched,
 and no Endorsed values are accepted.
 
 The `authorized-by` value in `measurement-map` in the stateful environment, if present,
-applies to all measurements in the triple, including `conditional-series-record` records.
+applies to reference value measurements in the triple, for `conditional-series-record` records.
+If the `authorized-by` value is provided in the `measurement-map` for endorsed values, they get added to the ACS.
```

Comment:
```suggestion
If the series `addition` entry contains `authorized-by` values, they are ignored.
```

Reply:
Note: authority semantics are still being discussed. It is safer for this triple to ignore them in the addition until the semantics for series authority is determined.

Reply:
I think since it's a non-extensible type, you can use a type like element-map (but a copy to not confuse internal and external representations) to drop the authorities. You can add them back with a spec update.

Reply:
ok, for now I will accept this suggestion , but really feel this is required. We will discuss the same in a separate PR